docs/application/web/get-started/tv/first-app.md

```diff
@@ -307,8 +305,7 @@ To run the application on a target device:
 
    3. Confirm that the application launches on the target device.
 
-   > **Note**
-   >
+   > [!NOTE]
```

Comment:
It looks like this note needs a line break.
![image](https://user-images.githubusercontent.com/9958374/71966535-24d08900-3245-11ea-9453-db515174dc53.png)